### PR TITLE
https://github.com/emprestes/chessgame/issues/22

### PR DESCRIPTION
**Pull request by Shutocan**
- Added toolchains.xml file into root directory of chess project.

In order to run the maven goals with the file in a different repository than .m2 you need to set a global property, e.g:

**mvn install -gt /path..../toolchains.xml -f pom.xml**

### CODE REVIEW

**Review by emprestes — CHANGES_REQUESTED** [review]
That's great and fast! Congrats!

My suggestion, considering a guideline and Gradle migration is ongoing, is just having this file in the `./src` folder instead of the root folder. What say you?

File: toolchains.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<toolchains>
+    <!-- JDK toolchains -->
+    <toolchain>
+        <type>jdk</type>
+        <provides>
+            <version>15</version>
+            <vendor>openJDK</vendor>
+        </provides>
+        <configuration>
+            <jdkHome>/usr/lib/jvm/java-15-oracle</jdkHome>
```
Comment (emprestes) [inline toolchains.xml:11]: This is a good point to include a comment like `<jdkHome><your path here />/jdk-15<jdkHome>`

Comment (Shutocan) [reply toolchains.xml]: Done

Comment (Shutocan) [reply toolchains.xml]: Also changed the file location

**Review by emprestes — APPROVED** [review]
@Shutocan , looks like we are ready to merge this PR, don't we?

**Pull request merged**